src/adapter/src/catalog/builtin.rs

```diff
@@ -1549,8 +1557,15 @@ pub const PG_CLASS: BuiltinView = BuiltinView {
     false AS relhasoids,
     -- MZ doesn't support options for relations
     NULL::pg_catalog.text[] as reloptions
-FROM mz_catalog.mz_objects
-JOIN mz_catalog.mz_schemas ON mz_schemas.id = mz_objects.schema_id
+FROM (
+    -- pg_class catalogs relations and indexes
```

Comment:
Kinda want this to be its own view if it is getting used twice? Unsure.

Reply:
Yeah the pg_ tables seems to include indexes in its definition of "relation", but we don't, which is the reason this comes up more than once. https://www.postgresql.org/docs/current/catalog-pg-class.html.
Happy to create a view for this, though I'm not sure what to call it - not sure why postgres calls it a "class".

Reply:
> not sure why postgres calls it a "class".

PostgreSQL once had aspirations of being an object-oriented database, where a table was a class and a row was an instance. It's all over the old docs, e.g. https://www.postgresql.org/docs/7.0/query27752.htm. They've mostly moved away from this idea, but it's still present in e.g. [table inheritance](https://www.postgresql.org/docs/current/ddl-inherit.html). 

> Happy to create a view for this, though I'm not sure what to call it

Is it not possible to rewrite `pg_attribute` in terms of this new-and-improved `pg_class` instead of `mz_objects` directly? Just glancing, so that might not work out.

Reply:
Ah that makes sense re: class.

> Is it not possible to rewrite pg_attribute in terms of this new-and-improved pg_class instead of mz_objects directly? Just glancing, so that might not work out.
No, unfortunately the `pg_attributes` query uses the `GlobalId` for a join, which is not included in `pg_class`.